apps/gp2-server/serverless.ts

```diff
@@ -59,6 +60,11 @@ const apiUrl = `https://${apiHostname}`;
 const currentRevision = process.env.CI_COMMIT_SHA;
 const nodeEnv = 'production';
 const sesRegion = process.env.GP2_SES_REGION!;
+const envRef = ['production', 'dev'].includes(stage) ? envAlias : `CI-${stage}`;
+
+const algoliaIndex = process.env.GP2_ALGOLIA_INDEX
```

Comment:
can you do...
```suggestion
const algoliaIndex = process.env.GP2_ALGOLIA_INDEX ?? `gp2-hub_${envRef}`;
```
?
I'm not sure I know what `'${env:....}'` is but it looks like it's a serverless way of referencing env vars which you are also doing with process.env so why complicate it